lightly/transforms/vicregl_transform.py

```diff
@@ -61,8 +67,8 @@ def __init__(
         self,
         global_crop_size: int = 224,
         local_crop_size: int = 96,
-        global_crop_scale: Tuple[int] = (0.2, 1.0),
-        local_crop_scale: Tuple[int] = (0.05, 0.2),
+        global_crop_scale: Tuple[float, float] = (0.2, 1.0),
+        local_crop_scale: Tuple[float, float] = (0.05, 0.2),
```

Comment:
Below in the code you set `self.transforms = ...`, I think you shouldn't do this and just use `transforms = ..` instead as you pass the variable to `super().__init__()` afterwards. The `ImageGridTransform` will then set `self.transforms`

Reply:
Same for the `VICRegViewTransform`

Reply:
I can't see the issue in the `ViewTransform`

Reply:
Ah sorry, you are right, the `ViewTransform` should store `self.transform`. All good!